lexers/v/vb.go

```diff
@@ -70,4 +74,10 @@ var VBNet = internal.Register(MustNewLexer(
 			Default(Pop(1)),
 		},
 	},
-))
+).SetAnalyser(func(text string) float32 {
+	if vbnetAnalyserRe.MatchString(text) {
```

Comment:
python version uses `re.MULTILINE` flag to apply this regex on the individual lines. But here you match it against the text as a whole, which does not result in the same behavior because of the `^` in the regex.